pkg/util/util.go

```diff
@@ -209,9 +209,15 @@ func GenFileURL(location string, os ...string) string {
 	}
 	urlPath := location
 	if currOS == WIN {
-		urlPath = "/" + strings.Replace(location, "\\", "/", -1)
+		urlPath = strings.Replace(location, "\\", "/", -1)
 	}
-	return "file://" + urlPath
+	if filepath.IsAbs(urlPath) || strings.Contains(urlPath, ":/") {
+		if currOS == WIN {
```

Comment:
This still doesn't look right. 
There is no reason why URI should be different based on OS.

Reply:
> There is no reason why URI should be different based on OS

TBH I didn't get your point, AFAIU we are using `file:///` URI across all the platform and yes we are adding `/` as a prefix on windows for a special case (when we have the absolute path from the root of different drive, I mean other than the current drive). In this special scenario the path would be like `C:\path\to\the\file`. However if we consider path from the root of current drive on windows or the absolute path on linux and macOS, it will always start with `/` for ex `/c/path/to/file` and hence combining `file://` with the path gives you the format `file:///`. Detailed absolute path description for windows has been described https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats . 
One more interesting fact for the special case is you won't be able to determine the path is absolute or not using `filepath.IsAbs` because it only checks the path has prefix `/`. 

> If we don't want to use URI format anymore, it is fine, but the output format needs to be consistent across all Operating Systems.

Well, I am not sure whether we should remove it or not because till now AFAIK we are using relative path for the json source field and I don't find any use case for the absolute. But if we have upcoming plans to use absolute in any case then keeping this would be helpful.

Reply:
You current code prints different output for win and Linux.
On windows path will be with `file://` on Linux it won't


All we need to to do fix the original issue is to make sure that the path aways starts with `file:///` (3 slashes)